doc/source/config.rst

```diff
@@ -37,6 +37,46 @@ several config files and pick from them using `-c`. If you only wish to control
 the specific tests that are to be run (and not their parameters) then using
 `-s` or `-t` on the command line may be more appropriate.
 
+Also you can configure bandit via
+`pyproject.toml <https://www.python.org/dev/peps/pep-0518/>`_ file. In this
+case you would explicitly specify the path to configuration via `-s` too.
```

Comment:
```suggestion
case you would explicitly specify the path to configuration via `-c` too.
```
The '-s' is for specifying which tests to skip within a configuration.  I believe you intended '-c' here, which is the argument to ask for the path to the configuration file.